addons/website_sale/static/src/css/website_sale.sass

```diff
@@ -96,14 +96,13 @@
         right: 0
         bottom: 0
         overflow: hidden
-        padding: 0 15px 24px 10px
+        padding: 0 15px 0 10px
         min-height: 56px
         border-top: 1px solid rgba(255,255,255,0.2)
         background: rgba(255, 255, 255, 0.75)
         z-index: 5
     .product_price
-        padding: 5px 0
-        position: absolute
+        padding: 0
```

Comment:
padding: 0 is useless, so either remove it or replace by padding-bottom: 5px to keep as before.